### DISCUSSION TITLE
Let the Installomator team know when your Apple Team ID changes

### DISCUSSION THREAD

**Discussion by ghost** [macadmins/python#56] [Ideas]
Hi there, since your project was added to Installomator two years ago, the Team ID was not kept up do date, causing deployments to fail with their software. I think it would be super cool if you would make sure to update them if it changes again. I've made a PR for the current ID.

Thanks for your time.

**Comment by erikng** [comment]
If someone is consuming our tools without our knowledge it's hard to inform everyone about it. That said, we shouldn't be changing the signing certs again. At least not for a long time.
